node/src/components/consensus/protocols/highway.rs

```diff
@@ -585,6 +585,21 @@ where
         }
     }
 
+    fn handle_latest_state_request(&mut self, sender: I) -> Vec<ProtocolOutcome<I, C>> {
+        let state = self.highway.state();
+
+        state
+            .panorama()
+            .iter_correct_hashes()
```

Comment:
…or should we also send evidence for the faulty entries?
(Not sure… I guess usually it'll be cited by some of the correct units anyway.)

Reply:
Yeah, and if that's a last evidence of fault it will be cited in the next era unit.

Reply:
Good question... It could be more correct, but probably not necessary.